### DISCUSSION TITLE
Are we ready for a release?

### DISCUSSION THREAD

**Discussion by mapkon** [Versent/saml2aws#995] [General]
Hello folks,

The last release was approximately 4 weeks ago, and there has been a raft of changes between then and now. For scale, here is the list of commits:

https://github.com/Versent/saml2aws/compare/v2.36.4...master

This should probably be a poll - I am ready to cut the release but I am gauging from the more experience folks if we feel we are in good stead to cut a release?

@wolfeidau @gliptak

**Comment by gliptak** [comment]
a (patch) release every few weeks is a good rhythm

**Reply by mapkon** [reply]
Yeah, getting to a stable cadence is the goal really, and we should work towards that in the long term.

That said, do you think we are in good shape to release?
